src/app/page.tsx

```diff
@@ -1,16 +1,15 @@
 import Contact from "./components/Contact/Contact";
 import HeroSection from "./components/HeroSection/HeroSection";
-import MemberSpotlight from "./components/MemberSpotlight/MemberSpotlight";
 import SlackInvite from "./components/SlackInvite/SlackInvite";
 import SpeakerInvite from "./components/SpeakerInvite/SpeakerInvite";
 import Sponsors from "./components/Sponsors/Sponsors";
 import WelcomeMessage from "./components/WelcomeMessage/WelcomeMessage";
 
 export default function Page() {
-  return <main>
+    return <main>
+    <title>BendJS</title>
```

Comment:
titles go in the head

Reply:
Can you elaborate please?

Reply:
yes sorry, title tags are meta data and therefor go in the head and not the body of the html document

In next13 this is handled with a metadata export

https://nextjs.org/docs/app/building-your-application/optimizing/metadata